Sources/ClientRuntime/Networking/Http/URLSession/URLSessionHTTPClient.swift

```diff
@@ -330,4 +413,71 @@ public enum URLSessionHTTPClientError: Error {
     case unresumedConnection
 }
 
+extension Bundle {
+    func certificate(named name: String) -> SecCertificate? {
+        guard let cerURL = self.url(forResource: name, withExtension: "cer"),
+              let cerData = try? Data(contentsOf: cerURL) else {
+            return nil
+        }
+        return SecCertificateCreateWithData(nil, cerData as CFData)
+    }
+
+    func identity(named name: String, password: String) -> SecIdentity? {
+        guard let p12URL = self.url(forResource: name, withExtension: "p12"),
+              let p12Data = try? Data(contentsOf: p12URL) else {
+            return nil
+        }
+
+        let options = [kSecImportExportPassphrase as String: password] as CFDictionary
+        var items: CFArray?
+        let status = SecPKCS12Import(p12Data as CFData, options, &items)
```

Comment:
`SecPKCS12Import`: The latest `aws-crt-swift` disabled the API for iOS, tvOS, and watchOS, as we dont really have a way to test the API on those platforms. Would you have a test plan to the feature with pkcs12 key on the platforms? Maybe I could add the tests to CRT as well and enable the API there. 

https://github.com/awslabs/aws-crt-swift/pull/246 <= PR that disabled `SecPKCS12Import`

Reply:
I agree there's no easy way to test on those platforms natively (would work via simulator for dev testing), I manually generated the p12 file using a combination of keychain access and openssl 1.1